scripts/drone/steps/lib.star

```diff
@@ -1,7 +1,7 @@
 load('scripts/drone/vault.star', 'from_secret', 'github_token', 'pull_secret', 'drone_token')
 
 grabpl_version = '2.5.5'
-build_image = 'grafana/build-container:1.4.4'
+build_image = 'grafana/build-container:1.4.5'
```

Comment:
@ashharrison90 @dsotirakis why isn't this version the same as the one in `scripts/build/ci-build/build-deploy.sh`

Reply:
Because `build-deploy.sh` is used for when you want to deploy the next `build-container` version. 
So by running `./build-deploy.sh` will deploy `grafana/build-container:1.4.6`.
I think this is misleading though since I've been asked again. I propose we delete this file.